hack/install.sh

```diff
@@ -17,9 +17,9 @@ kubectl apply -f manager/config/prod/deployment_configmap.yaml -n $KUBE_NAMESPAC
 make cluster-prepare
 
 # Install third party components
-WITHOUT_VAULT || make -C third_party/vault deploy
-WITHOUT_EGERIA || make -C third_party/egeria deploy
-WITHOUT_OPA || make -C third_party/opa deploy
+$WITHOUT_VAULT || make -C third_party/vault deploy
```

Comment:
@tomersolomon1 you are running "make cluster-prepare" which runs "$(MAKE) -C third_party/vault deploy"
See line 28 at Makefile - https://github.com/IBM/the-mesh-for-data/blob/master/Makefile#L28
there is a duplication, one that always install Vault, the other on condition.

Reply:
Suggestion - remove Vault installation from the `cluster-prepare` phase, so it's installed (only) by the installation script - and keep it optional.
Anyhow, if we're talking about the `cluster-prepare` phase, why we need it to create `registry` secret?
https://github.com/IBM/the-mesh-for-data/blob/master/Makefile#L27
https://github.com/IBM/the-mesh-for-data/blob/master/third_party/registry/deploy.sh#L18

Is it possible this is a leftover from the time before moving to public docker registry?

Reply:
Let's open a new issue for these

Reply:
See https://github.com/IBM/the-mesh-for-data/issues/83